hasher-matcher-actioner/hmalib/lambdas/api/api_root.py

```diff
@@ -172,6 +173,23 @@ def process_s3_event(event: dict) -> None:
     ),
 )
 
+
+app.mount(
+    "/lcc/",
+    get_lcc_api(
+        # Not sure which of these are needed. Followed similar format above
```

Comment:
You can remove everything that you don't need. I feel beyond the env var for the EFS mount path and the bucket width, you'll not need anything.

That's for @franklin-phan to figure out. You can remove everything.